hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/Constants.java

```diff
@@ -32,4 +32,6 @@
   public static final String ZK_MAX_RETRIES = "fs.hboss.sync.zk.sleep.max.retries";
 
   public static final String CONTRACT_TEST_SCHEME = "fs.contract.test.fs.scheme";
+
+  public static final String WAIT_INTERVAL_WARN = "fs.hboss.lock-wait.interval.warning";
```

Comment:
This configuration name is defined but it is not read from the configuration. Initialize waitIntervalWarn from the config and fall back to the default value.

Reply:
It is read on the impl class, _ZKTreeLockManager.initialize_ method, as _TreeLockManager_ is abstract and doesn't have an init/constructor method.

Reply:
I missed that part, already marked ZKTreeLockManager as Viewed when I searched for the config usage. 🙂